### PR TITLE
Update issue templates and contribution guidelines

### PR DESCRIPTION
**Pull request by denisok**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a report to help us improve the exporter
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. 
+If the issue relates to PMM, please open [JIRA PMM](https://jira.percona.com/projects/PMM/issues) issue instead of GH issue.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. what parameters passed to `mongodb_exporter`
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
1. what parameters are being passed to `mongodb_exporter`
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+2. describe triggers to reproduce the issue
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: What are triggers? Maybe the steps?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Please provide logs when issue occurs
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
Please provide logs relevant to the issue
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+If the improvement is related to PMM, please open [PMM project](https://jira.percona.com/projects/PMM/issues) feature instead of GH feature request.
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]:
```suggestion
A clear and concise description of what you expect to happen.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]:
```suggestion
Add any other context or screenshots related to the feature request here.
```

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## The ABCs
+
+* **A**ssume good intentions.
+* **B**e nice to each other - being civil and respectful goes a long way.
+* **C**riticize code, tools, other types of contributions, and situations - not people and their personalities.
+
+## Rules
+
+We respect all members of the Percona Community - that involves everyone’s contributions in code, text, visuals, and chat. Your contributions might take place in one of our GitHub organizations, on Jira, on the forums, Slack, or elsewhere.
```
Comment (atymchuk) [inline CODE_OF_CONDUCT.md:13]: Community has no access to Slack. Contributing elsewhere means we may never be able to see the contribution.

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+We all are committed to participating in a community where abuse and harrassment is neither tolerated nor accepted. If you want to give feedback, remember to criticize the code or other type of contribution, not the person.
```
Comment (atymchuk) [inline CODE_OF_CONDUCT.md:15]:
```suggestion
We all are committed to participating in a community where abuse and harassment is neither tolerated nor accepted.
```
Part of it is a duplicate of ABCs.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -37,6 +40,21 @@ As a general rule of thumb, please try to create bug reports that are:
...
+`main` is a main branch where all the changes are merged **first**.
+
+`release-x.y` are the release branches and are forked from `main` to prepare and test the release. Release branch stays open to get critical patches and to release another _patch_ release. Release branch closes (no merges happens) just after new minor release is created.
+
+Each release is tagged with `vx.y.z` tag.
+
+
+Please submit your changes against `main` branch, if the fix is needed for the patch or minor release - please ask maintainers to cherry pick it into the release branch.
+
 ## Setup your local development environment
```
Comment (atymchuk) [inline CONTRIBUTING.md:58]:
```suggestion

```
We can remove those two and add them later once ready.

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (denisok) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: steps are OK

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (denisok) [reply .github/ISSUE_TEMPLATE/feature_request.md:21]: @atymchuk  BTW all that is standard GH template

File: CODE_OF_CONDUCT.md (hunk 8 above)
Comment (denisok) [reply CODE_OF_CONDUCT.md:15]: Could you please update the source of this one? I just would copy paste from thee the new version: https://github.com/percona/community/blob/main/content/contribute/coc.md

File: CONTRIBUTING.md (hunk 9 above)
Comment (denisok) [reply CONTRIBUTING.md:58]: didn't get it. Later in the chapter there is a desc on how to setup the env

Comment (atymchuk) [reply CONTRIBUTING.md:58]: > didn't get it. Later in the chapter there is a desc on how to setup the env

Sorry, didn't notice there was a section. Pls ignore.

**Review by percona-csalguero — APPROVED** [review]

**Comment by atymchuk** [comment]
@denisok Just landed two more improvements ) Please see if you can integrate them, otherwise good to go.

**Review by atymchuk — APPROVED** [review]

**Pull request merged**